driver/normalizer/normalizer.go

```diff
@@ -772,3 +770,180 @@ func (op dropNils) Check(st *State, n nodes.Node) (bool, error) {
 func (op dropNils) Construct(st *State, n nodes.Node) (nodes.Node, error) {
 	return op.op.Construct(st, n)
 }
+
+var (
+	typeGroup     = uast.TypeOf(uast.Group{})
+	typeFuncGroup = uast.TypeOf(uast.FunctionGroup{})
+)
+
+// triviaField specified a field with an array to put trivias into.
+var triviaField = map[string]string{
+	"Block":           "Statements",
+	"CompilationUnit": "Members",
+}
+
+// opMoveTrivias cuts trivia nodes from LeadingTrivia/TrailingTrivia fields
+// and wraps the node into uast:Group that contains those trivias.
+type opMoveTrivias struct {
+	sub Op
+}
+
+func (op opMoveTrivias) Kinds() nodes.Kind {
+	return nodes.KindObject
+}
+
+func (op opMoveTrivias) Check(st *State, n nodes.Node) (bool, error) {
+	obj, ok := n.(nodes.Object)
+	if !ok {
+		return false, nil
+	}
+	cloned := false
```

Comment:
Maybe `modified` is a better name here? (We cloned it because it's changed, but the change appears to be the important part of the meaning)